### DISCUSSION TITLE
I discover a species that is quite strong.

### DISCUSSION THREAD

**Discussion by wmCmo** [MaxRobinsonTheGreat/LifeEngine#134] [General]
So I've been playing with the simulator and found this species. It is quite strong for the following reasons:
- It has a mechanism that lets it spread super fast.
- It mutates but will remain the identical look (It looks like a berry to me.)
- They create a wall of killer cells while spreading, making a large organism have a tough time getting through.
- They survived even when living with the smallest cruelest hunter.
![image](https://user-images.githubusercontent.com/51499433/172150557-8eda7732-c49f-4710-9dee-5c022ae1e9d9.png)

**Comment by AxoWasTaken** [comment]
I put one in an environment filled with only unnatural-engineered organisms and in seconds it spread to the other side of the world. Its very strong!

**Reply by SahilKDas** [reply]
Can you share the JSON of the creature so we can use it?
